Sources/JetStream/Stream.swift

```diff
@@ -35,6 +53,203 @@ public class Stream {
         }
     }
 
+    /// Retrieves a raw message from stream.
+    ///
+    /// - Parameters:
+    ///   - sequence: The sequence of the message in the stream.
+    ///   - subject: The stream subject the message should be retrieved from.
+    ///     When combined with `seq` will return the first msg with seq >= of the specified sequence.
+    ///
+    /// - Returns ``RawMessage`` containing message payload, headers and metadata.
+    ///
+    /// - Throws:
+    ///   - ``JetStreamRequestError`` if the request was unsuccesful.
+    ///   - ``JetStreamError`` if the server responded with an API error.
+    public func getMsg(sequence: UInt64, subject: String? = nil) async throws -> RawMessage? {
```

Comment:
we have `getMsg` and that returns `RawMessage`. I think we should use `Message` consistently across library.